R/run_stress_test.R

```diff
@@ -353,14 +345,25 @@ st_read <- function(dir, asset_type) {
   return(out)
 }
 
+st_read_agnostic <- function(dir, asset_type) {
```

Comment:
asset_type is not a necceasary input arg to asset_read_agnositc (at least as of now)

Reply:
Thanks for noting that! Done.